INSTALL.txt

```diff
@@ -2,20 +2,94 @@ Installers for Windows and Mac OS X and tarballs for Linux can be found at:
+    # UMS requires a Java Virtual Machine to run. Which version depends on which binary package was downloaded. UMS for Java 7
+    would need version 7 or newer, UMS for Java 8 would need version 8 or newer. The most common JVM's to use on Linux is
+    openjdk. Which, if any, of the packages that's available depends on your distro, it's version and what sources are enabled.
+
+    # Install JVM on Debian/Ubuntu and derivates:
+      Debian 8 need to use backports to install openjdk-8 at the time of writing. It's not possible to cover all the sources
+      needed for the different versions in this document. If the package isn't found, make sure to run `apt-get update` to
+      make sure the sources are refreshed.
+      # To install openjdk-7 run:
+      sudo apt-get install openjdk-7-jre
+      # To install openjdk-8 run:
+      sudo apt-get install openjdk-8-jre
+
+    # Install JVM on Fedora, OpenSUSE, Mandriva, Red Hat/Centos and derivates: The epel repository might be needed on some
+    platforms.
+      # To install openjdk-7 run:
+      sudo yum install java-1.7.0-openjdk
+      # To install openjdk-8 run:
+      sudo yum install java-1.8.0-openjdk
+
+    # Install JVM on Arch:
+      # To install openjdk-7 run:
+      sudo pacman -S jre7-openjdk
+      # To install openjdk-8 run:
+      sudo pacman -S jre8-openjdk
+
+    # UMS use MediaInfo to get information about media files. It can be run without it, but media handling will suffer as UMS
+    won't have a detailed information available.
+    # Install MediaInfo on Debian/Ubuntu and derivates:
+    sudo apt-get install mediainfo
+    # Install MediaInfo on Fedora, OpenSUSE, Mandriva, Red Hat/Centos and derivates:
+    sudo yum install mediainfo
+    # Install MediaInfo on Arch (community repository must be enabled):
+    sudo pacman -S mediainfo
+
+    # You can optionally install dcraw (to decode raw images from digital cameras) and VLC (to use as a transcoding engine).
+    Just omit one if you don't want that.
+    # Install dcraw and VLC on Debian/Ubuntu and derivates:
+    sudo apt-get install dcraw vlc
+    # Install dcraw and VLC on Fedora, OpenSUSE, Mandriva, Red Hat/Centos and derivates:
+    sudo yum install dcraw vlc
+    Note: Before installing VLC, you'll need to enable the free RPM Fusion repository in your system:
+    sudo yum install --nogpgcheck https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E%fedora).noarch.rpm
+    # Install VLC on Fedora 22 and later:
+    sudo dnf -y install vlc
+    Note: Before that, you'll need to enable the free RPM Fusion repository in your system:
+    sudo dnf install http://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm
+    # Install dcraw and VLC on Arch (extra repository must be enabled):
+    sudo pacman -S dcraw vlc
+
+    # UMS uses tsMuxeR as a transcoding engine. tsMuxeR is 32 bit only, so installation of some 32 bit libraries are necessary
```

Comment:
Maybe add a note about tsMuxeR being disabled by default, since this is potentially unnecessary work for a user installing UMS